doc/getting_started.rst

```diff
@@ -0,0 +1,277 @@
+``pycd`` in your terminal. You will need to do this for each terminal
+window you use.
+
+Installing Dependencies
+=======================
+
+Below, we list the dependencies Vivarium Core requires, how to check
+whether you have them, how to install them, and in some cases, how to
+set them up for Vivarium Core.  Make sure you have each of them
+installed.
+
+Python 3
+--------
+
+We wrote Vivarium Core in Python 3. While we did also write it to be
+Python-2-compatible, `the Python Software Foundation has deprecated
+Python 2 <https://www.python.org/dev/peps/pep-0373/>`_, so we strongly
+recommend using Python 3. We also no longer guarantee compatibility with
+Python 2.
+
+*Check Installation*
+
+.. code-block:: console
+
+    $ python --version
+    Python <version>
+
+Make sure you see a version at least 3.6.
+
+*Install*
+
+Download the latest installer from the `Python download page
+<https://www.python.org/downloads/>`_
+
+MongoDB
+-------
+
+We use a MongoDB database to store the data collected from running
+simulations. This can be a remote server, but for this guide we will
+run a MongoDB server locally.
+
+*Check Installation*
+
+.. code-block:: console
+
+    $ mongod --version
+    db version v4.2.3
+    ...
+
+Make sure you see a version at least 4.2.
+
+*Install*
+
+If you are on macOS, you can install MongoDB using `Homebrew
+<https://brew.sh>`_. You will need to add the MongoDB tap following the
+instructions `here <https://github.com/mongodb/homebrew-brew>`_.
+
+If you are on Linux, see the MongoDB documentation's `instructions
+<https://docs.mongodb.com/manual/administration/install-on-linux/>`_.
+
+*Setup*
+
+You can get a MongoDB server up and running locally any number of ways.
+Here is one:
+
+#. Create a folder ``vivarium_work/mongodb``. This is where MongoDB will
+   store the database We store the database here instead of at the
+   default location in ``/usr/local/var/mongodb`` to avoid permissions
+   issues if you are not running as an administrator.
+#. Make a copy of the ``mongod`` configuration file so we can make
+   changes:
+
+   .. code-block:: console
+
+      $ cp /usr/local/etc/mongod.conf vivarium_work/mongod.conf
+
+   Note that your configuration file may be somewhere slightly
+   different. Check the MongoDB documentation for your system.
+#. In ``vivarium_work/mongod.conf`` change the path after ``dbPath:`` to
+   point to ``vivarium_work/mongodb``.
+#. Create a shell script ``vivarium_work/mongo.sh`` with the following
+   content:
+
+   .. code-block:: bash
+
+      #!/bin/bash
+
+      mongod --config mongodb.conf
+
+#. Make the script executable:
+
+   .. code-block:: console
+
+        $ chmod 700 vivarium_work/mongo.sh
+
+   Now you can launch MongoDB by running this script:
+
+   .. code-block:: console
+
+        $ vivarium_work/mongo.sh
+
+.. todo:: Use ``py -m agent.boot --host ip.to.remote.cluster:9092``
+    for remote Kafka services
+
+GNU Linear Programming Kit (GLPK)
```

Comment:
I believe this is only needed for ```vivarium-cell```... Points to the issue of having multiple possible ```getting_started``` docs for different repos.

Reply:
removed in f1f742fff55012cf7398b512f3ef1928fbdfbc14